kolibri/plugins/facility/assets/src/views/users/common/UsersTable.vue

```diff
@@ -158,10 +158,9 @@
       @close="closeModal"
     />
 
-    <DeleteUserModal
+    <MoveToTrashModal
       v-if="modalShown === Modals.DELETE_USER"
-      :id="userToChange.id"
-      :username="userToChange.username"
+      :selectedUsers="new Set([userToChange.id])"
```

Comment:
(Non blocking) If we create a new set inline in the template, we will be creating a new set on every re-render and slightly increasing the memory usage. A better approach can be to create a computed property that only gets recalculated when the rective prop is changed, and then pass that computed property here, so we avoid creating new sets on every re-render.